projects/libusb.info/package.yml

```diff
@@ -0,0 +1,82 @@
+distributable:
+  url: https://github.com/libusb/libusb/releases/download/v{{ version }}/libusb-{{ version }}.tar.bz2
+  strip-components: 1
+
+versions:
+  github: libusb/libusb/releases/tags 
+  strip: /^v/
+
+build:
+  dependencies:
+    tea.xyz/gx/cc: c99
+    tea.xyz/gx/make: '*'
+    gnu.org/autoconf: ^2
+  script: |
+    ./configure $ARGS
+    make install
```

Comment:
Don't forget `-j {{ hw.concurrency }}` to take advantage of parallelizable builds.